apps/example/app.json

```diff
@@ -48,6 +48,8 @@
         "expo-build-properties",
         {
           "ios": {
+            // Minimum iOS version we support
+            "deploymentTarget": "15.1",
```

Comment:
This is the current minimum supported by expo.

In the wallet we were at 15.0, but all devices that can install iOS 15.0 can also install 15.1 so shouldn't affect users.